src/bloom-filter.jl

```diff
@@ -38,11 +39,12 @@ end
 
 function BloomFilter(mmap_handle::IOStream, capacity::Int, bits_per_elem::Int, k_hashes::Int)
     n_bits = capacity * bits_per_elem
-    mb = mmap_bitarray((n_bits, 1), mmap_handle)
+    # mb = mmap_bitarray((n_bits, 1), mmap_handle)
```

Comment:
Generally all looks good to me! Small nit: Mind removing the old `mmap_bitarray` calls?